backend/ASTU-backend-group-1/config.yaml

```diff
@@ -5,3 +5,4 @@ database:
 email:
   key: "nenbydebhwkqanpf"
 port: "http://localhost:8000/"
+jwt: "starterproject"
```

Comment:
Instead of having one big field called jwt have it as a parent and add the other children inside it to make it more readable and configurable